pvlib/tests/test_modelchain.py

```diff
@@ -719,7 +719,9 @@ def test_run_model_with_weather_noct_sam_temp(sapm_dc_snl_ac_system, location,
     assert m_noct_sam.call_count == 1
     assert_series_equal(m_noct_sam.call_args[0][1], weather['temp_air'])
     assert_series_equal(m_noct_sam.call_args[0][2], weather['wind_speed'])
-    assert not mc.results.ac.empty
+    # check that effective_irradiance was used
+    assert m_noct_sam.call_args.kwargs == {
+        'effective_irradiance': mc.results.effective_irradiance}
```

Comment:
```suggestion
    assert m_noct_sam.call_args[1] == {
        'effective_irradiance': mc.results.effective_irradiance}
```

Reply:
`call_args.kwargs` was added in python 3.8 (I think), so you have to use the old style look up for the older supported versions. If this doesn't work try 

```
assert_series_equal(m_noct_sam.call_args[1]['effective_irradiance'], mc.results.effective_irradiance)
```

Reply:
Thanks. I was puzzled by the disconnect between pytest docs and the results, didn't think to check versions.